Index.php

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
```

Comment:
there's no PHP in this file, it should be an html file

Reply:
There is no Php for now , but I'm gonna add some for story 2